### PR TITLE
Adopt Sabre 4.7.0 in the side service test suite

### PR DESCRIPTION
**Pull request by vttranlina**
## Summary by CodeRabbit

* **New Features**
  * Added configuration option to enable public calendar visibility settings.

* **Tests**
  * Expanded test coverage for event indexing with improved edge-case handling.
  * Optimized test execution timeouts for better performance.
  * Removed obsolete test cases.

* **Chores**
  * Updated service images to stable releases.
  * Simplified provisioning configuration for streamlined deployment.

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]

File: calendar-amqp/src/test/java/com/linagora/calendar/amqp/EventIndexerConsumerTest.java [hunk 1]
```diff
@@ -517,7 +517,7 @@ void shouldUpdateRecurrenceEvent() {
             ORGANIZER;CN=John1 Doe1:mailto:{organizer}
             ATTENDEE;PARTSTAT=NEEDS-ACTION;RSVP=TRUE;ROLE=REQ-PARTICIPANT;CUTYPE=INDIVI
              DUAL;CN=John2 Doe2;SCHEDULE-STATUS=1.1:mailto:{attendee1}
-            ATTENDEE;PARTSTAT=ACCEPTED;RSVP=FALSE;ROLE=CHAIR;CUTYPE=INDIVIDUAL:{organizer}
+            ATTENDEE;PARTSTAT=ACCEPTED;RSVP=FALSE;ROLE=CHAIR;CUTYPE=INDIVIDUAL:mailto:{organizer}
```
Comment (chibenwa) [inline calendar-amqp/src/test/java/com/linagora/calendar/amqp/EventIndexerConsumerTest.java]: Please have at least one test to ensure we support the old syntax without mailto

**Comment by vttranlina** [comment]
Can’t reproduce the failed test locally.
It looks like an AMQP pub/sub message ordering issue.
Need to take more time to investigate

File: calendar-amqp/src/test/java/com/linagora/calendar/amqp/EventIndexerConsumerTest.java (hunk 1 above)
Comment (chibenwa) [reply calendar-amqp/src/test/java/com/linagora/calendar/amqp/EventIndexerConsumerTest.java]: I want no regression in behaviours.
Please add the tet I asked for.

**Comment by chibenwa** [comment]
Please @vttranlina 

 -> Remove the failing test
 -> Remove the System.out.println
 
 And let's merge this timely.
 
 Thanks.

File: calendar-amqp/src/test/java/com/linagora/calendar/amqp/AlarmEventUpdateTest.java [hunk 2]
```diff
@@ -456,57 +456,6 @@ void shouldUpdateAlarmTimeOfAttendeeWhenOrganizerChangeStartDateLater() {
         });
     }
 
-    @Test
-    void shouldUpdateIcsPayloadOfAttendeeWhenOrganizerChangeEvent() {
```
Comment (chibenwa) [inline calendar-amqp/src/test/java/com/linagora/calendar/amqp/AlarmEventUpdateTest.java]: This test is legit and tests https://github.com/linagora/esn-sabre/issues/227

IMO let's not remove it.

Let's mark it @Disabled and reference the ticket
